python/lsst/ts/Dome/dome_csc.py

```diff
@@ -0,0 +1,142 @@
+# import asyncio
+import pathlib
+from lsst.ts import salobj
+
+__all__ = ['DomeCsc']
+
+
+class DomeCsc(salobj.ConfigurableCsc):
+    """
+    Upper level Commandable SAL Component to interface with the LSST Dome lower level components.
+    """
+
+    def __init__(
+            self,
+            config_dir=None,
+            initial_state=salobj.State.STANDBY,
+            simulation_mode=0,
+    ):
+        schema_path = (
+            pathlib.Path(__file__)
+            .resolve()
+            .parents[4]
+            .joinpath("schema", "Dome.yaml")
+        )
+
+        self.reader = None
+        self.writer = None
+        self.config = None
+
+        self.mock_ctrl = None  # mock controller, or None if not constructed
+        super().__init__(
+            name="DCS",
+            index=0,
+            schema_path=schema_path,
+            config_dir=config_dir,
+            initial_state=initial_state,
+            simulation_mode=simulation_mode,
+        )
+
+    async def do_Dome_command_crawlAz(self, id_data):
+        """
+        Crawl AZ
+        """
+        pass
```

Comment:
You may want to go ahead and add the check

```
self.assert_enable()
```

on all commands.

Reply:
Thanks, done.